sources/se/municipality_of_kristinehamn.json

```diff
@@ -0,0 +1,29 @@
+{
+    "coverage": {
+        "country": "se",
+        "city": "Kristinehamn"
+    },
+    "data": "http://kartor-va.kristinehamn.se/arcgis/rest/services/Bakgrundskartor/Bakgrundskarta_fargad_enkel/MapServer/112",
+    "type": "ESRI",
+    "conform": {
+        "type": "geojson",
+        "number": {
+            "function": "regexp",
+            "field": "Name",
+            "pattern": "([0-9]+([A-Z])?)$"
```

Comment:
Might be an issue with the inner capture group. Also, shouldn't this be identical to the regex below just with different captures? I'd try:  "^(?:.*?)(?: ([0-9]+[A-Z]?))?$"